docs/dev/projects.md

```diff
@@ -0,0 +1,44 @@
+<!--Copyright (C) 2021 SuperDARN Canada, University of Saskatchewan 
+Author(s): Marina Schmidt 
+Modifications:
+
+Disclaimer:
+pyDARN is under the LGPL v3 license found in the root directory LICENSE.md 
+Everyone is permitted to copy and distribute verbatim copies of this license 
+document, but changing it is not allowed.
+
+This version of the GNU Lesser General Public License incorporates the terms
+and conditions of version 3 of the GNU General Public License, supplemented by
+the additional permissions listed below.
+-->
+
+# GitHub Projects
+
+[Projects on GitHub](https://docs.github.com/en/github/managing-your-work-on-github/about-project-boards) are an easy way to organize task for a general or specific project/feature. 
+
+## Getting Started 
+
+Click on [New project](https://github.com/SuperDARN/pydarn/projects/new) to start with a new project. 
+If you want to see other projects click on [pydarn projects](https://github.com/SuperDARN/pydarn/projects) for examples of the types of projects. 
+
+When selecting a template it is recommended to use [Automated Kanban](https://docs.github.com/en/github/managing-your-work-on-github/about-automation-for-project-boards) as it will move tasks throughout the workflow. 
```

Comment:
```suggestion
When selecting a template it is recommended to use [Automated Kanban](https://docs.github.com/en/github/managing-your-work-on-github/about-automation-for-project-boards) as it will move tasks through the workflow automatically. 
```